packages/browser-sdk/README.md

```diff
@@ -172,6 +174,43 @@ by down-stream clients, like the React SDK.
 Note that accessing `isEnabled` on the object returned by `getFeatures` does not automatically
 generate a `check` event, contrary to the `isEnabled` property on the object returned by `getFeature`.
 
+### Feature Overrides
+
+You can override feature flags locally for testing purposes using `setFeatureOverride`:
+
+```ts
+// Override a feature to be enabled
+bucketClient.setFeatureOverride("huddle", true);
+
+// Override a feature to be disabled
+bucketClient.setFeatureOverride("huddle", false);
+
+// Remove the override
+bucketClient.setFeatureOverride("huddle", null);
+
+// Get current override value
+const override = bucketClient.getFeatureOverride("huddle"); // returns boolean | null
+```
+
+Feature overrides are persisted in `localStorage` and will be restored when the page is reloaded.
+
+### Feature Updates
+
+You can listen for feature updates using `onFeaturesUpdated`:
+
+```ts
+// Register a callback for feature updates
+const unsubscribe = bucketClient.onFeaturesUpdated(() => {
+  console.log("Features were updated");
+});
+
+// Later, stop listening for updates
+unsubscribe();
+```
+
+> [!NOTE]
+> Note that the callback may be called even if features haven't actually changed.
```

Comment:
I don't think these notes work on GitBook unfortunately :/

Reply:
Sucks :( Can't find something in common.

Reply:
And pedantic I went... added an awk script to convert from GitHub to GitBook format so now we can use GitHub format and it'll look good in both places.

Reply:
hah, nice!